### PR TITLE
Changed model to store time for using UTC timezone 

### PR DESCRIPTION
**Pull request by ayushdevelop**
# functionalities Added or Changed

- changed the `Prisma` model to store UTC time zone
- made changes in the backend to convert `UTC` time to `IST`
 - scheduled standups should run at the time set according to `IST`

### CODE REVIEW

File: server/src/services/checkStandupAvailaibility.ts [hunk 1]
```diff
@@ -75,8 +75,20 @@ export const scheduledStandups = () => {
 };
 
 export const standupInThisHour = async () => {
+  // converting UTC time to 24H format IST time
+  const hour = new Date(`${dayjs.utc().format()}`)
```
Comment (SujayPrabhu96) [inline server/src/services/checkStandupAvailaibility.ts:79]: Console this variable using logger

File: server/src/services/checkStandupAvailaibility.ts [hunk 2]
```diff
@@ -75,8 +75,20 @@ export const scheduledStandups = () => {
...
+    .toLocaleString("en-IN", { hour12: false, timeZone: "Asia/Kolkata" })
+    .substr(12)
+    .split(":")[0];
+
+  // comparing time retrieved from database and current time to check available standups for current hour
   standupsAvailableThisHour = await standupsAvailableToday.filter(
-    (standup: any) => standup.localTime?.split(":")[0] === thisHour
+    (standup: any) =>
+      new Date(`${standup.originalTime}`)
```
Comment (SujayPrabhu96) [inline server/src/services/checkStandupAvailaibility.ts:87]: Also this

Comment (SujayPrabhu96) [reply server/src/services/checkStandupAvailaibility.ts:87]: Log
```
new Date(`${standup.originalTime}`).toLocaleString("en-IN", { hour12: false, timeZone: "Asia/Kolkata" })
```

File: server/src/services/checkStandupAvailaibility.ts (hunk 1 above)
Comment (ayushdevelop) [reply server/src/services/checkStandupAvailaibility.ts:79]: okay

**Pull request merged**